### PR TITLE
Internationalization of old strings

### PR DESCRIPTION
**Pull request by jafowler**
This is mostly just the changes needed in each spec. Haven't even started looking at the components yet.

### CODE REVIEW

File: src/parser/deathknight/blood/modules/features/BoneShield.js [hunk 1]
```diff
@@ -43,7 +45,7 @@ class BoneShield extends Analyzer {
     when(this.uptimeSuggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => suggest('Your Bone Shield uptime can be improved. Try to keep it up at all times.')
```
Comment (MartijnHols) [inline src/parser/deathknight/blood/modules/features/BoneShield.js:46]: These are also important to internationalize

File: src/parser/deathknight/blood/modules/features/DancingRuneWeapon.js [hunk 2]
```diff
@@ -74,7 +76,7 @@ class DancingRuneWeapon extends Analyzer {
     when(this.SuggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => suggest(<>Avoid casting spells during <SpellLink id={SPELLS.DANCING_RUNE_WEAPON.id} /> that don't benefit from the coppies such as <SpellLink id={SPELLS.BLOODDRINKER_TALENT.id} /> and <SpellLink id={SPELLS.DEATH_AND_DECAY.id} />. Check the cooldown-tab below for more detailed breakdown.{this.goodDRWSpells}</>)
```
Comment (MartijnHols) [inline src/parser/deathknight/blood/modules/features/DancingRuneWeapon.js:77]: These are also important to internationalize with <Trans id="x">

**Review by MartijnHols — APPROVED** [review]

**Comment by MartijnHols** [comment]
Thanks!

File: src/parser/deathknight/blood/modules/features/BoneShield.js (hunk 1 above)
Comment (jafowler) [reply src/parser/deathknight/blood/modules/features/BoneShield.js:46]: Do these just get wrapped in Trans tags?

Comment (MartijnHols) [reply src/parser/deathknight/blood/modules/features/BoneShield.js:46]: It doesn't matter for strings, though using Trans everywhere might be easier.

If you can fix the conflicts for this PR I'd like to merge this before that though.

Comment (jafowler) [reply src/parser/deathknight/blood/modules/features/BoneShield.js:46]: Done!

**Pull request merged**